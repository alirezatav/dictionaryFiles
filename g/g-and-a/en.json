{"word[0]":"G & A","definition[0]":"written abbreviation for GENERAL AND ADMINISTRATIVE EXPENSES"}