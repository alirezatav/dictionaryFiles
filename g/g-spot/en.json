{"word[0]":"G-spot","definition[0]":"the place in a womanâ€™s vagina where she is thought to feel the most sexual pleasure"}