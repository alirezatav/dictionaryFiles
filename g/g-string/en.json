{"word[0]":"G-string","definition[0]":"a very small piece of underwear that covers only the sexual organs"}