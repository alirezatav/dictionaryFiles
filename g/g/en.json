{"word[0]":"G","plural[0]":"G’s","definition[0]":"the seventh letter of the English alphabet","definition[1]":"the fifth note in the musical scale of C major or the musical key based on this note","definition[2]":"used to describe a film that has been officially approved as suitable for people of any age","definition[3]":"a unit for measuring the force caused by gravity on an object as it starts to move faster and faster","example[0]":"Astronauts endure a force of several G’s during take-off.","example[1]":"high g-forces","definition[4]":"a grand (=$1,000)","definition[5]":"a written abbreviation of  gram or grams","definition[6]":"written abbreviation for GRAM(S)","definition[7]":"abbreviation for GRAND"}