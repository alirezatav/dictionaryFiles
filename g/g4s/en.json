{"word[0]":"G4S","definition[0]":"an international security organization, which provides many different services, including carrying money and valuable goods, and taking prisoners in guarded vehicles to and from law courts."}