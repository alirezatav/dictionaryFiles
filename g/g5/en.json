{"word[0]":"G5"}