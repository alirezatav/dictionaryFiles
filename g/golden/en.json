{"word[0]":"golden","definition[0]":"having a bright yellow colour like gold","example[0]":"golden hair","example[1]":"golden sand","definition[1]":"a golden period of time is one of great happiness or success","example[2]":"the golden age of radio","example[3]":"the golden years of childhood","note[0]":"golden years/days etc","definition[2]":"made of gold","example[4]":"a golden crown"}